pymove/utils/mem.py

```diff
@@ -2,9 +2,13 @@
 
 import json
 import os
-import pwd
+try:
+    import pwd
+    import resource
+except (ModuleNotFoundError, ImportError)  as e:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'e' is assigned to but never used (F841)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5497268)